manticore/ethereum/manticore.py

```diff
@@ -390,7 +390,7 @@ def contract_accounts(self):
     def get_account(self, name):
         return self._accounts[name]
 
-    def __init__(self, workspace_url: str = None, policy: str = "random"):
+    def __init__(self, plugins=None, **kwargs):
```

Comment:
This is definitely a user-facing API change, but it might be okay because I can't think of a time I've ever seen anybody use the workspace_url or policy parameters, for EVM or otherwise. If anyone has objections, we can, of course, tweak this so that workspace_url and policy will still be in their correct position. 

If nothing else, the `param` docstring items are still outdated and should be changed

Reply:
Also this is walking toward removing "register_plugin" et.al. and forcing it to be registered at Manticore creation.

`policy` is actually ignored and `workspace_url` is more a parameter of the parent class